docs/core/compatibility/sdk/6.0/implicit-namespaces.md

```diff
@@ -57,17 +57,19 @@ This change was made to reduce the number of boilerplate `using` directives that
 
 For most users, you need take no action. However, this change can cause type-name conflicts with the namespaces that are implicitly included. If that happens, modify the list of `global using` directives or fully qualify your type references. You can modify the list of `global using` directives in several ways:
 
-- Disable the feature completely by setting `<DisableImplicitNamespaceImports>` to `true` in the project file. For more information, see [DisableImplicitNamespaceImports](../../../project-sdk/msbuild-props.md#disableimplicitnamespaceimports).
+- Disable the feature completely by setting `<ImplicitUsings>` to `false` in the project file. For more information, see [ImplicitUsings](../../../project-sdk/msbuild-props.md#implicitusings).
```

Comment:
I'm not sure that we should update this specific article. It is specific to .NET 6 Preview 7, as noted in the intro, and so it refers to the older syntax (that was changed in RC 1).

Reply:
I don't think it would make sense to leave this stuck at Preview 7.  It's effectively an article about changes introduced in .NET 6, and after GA it will cover what's different from 5.0 to 6.0.

Reply:
It's more complicated than that. Preview 7 turned on implicit usings for new and existing projects. RC 1 introduced a _new_ breaking change where implicit usings were disabled for existing projects. So that single breaking change article really is specific to Preview 7, and will be deleted once .NET 6 GAs. The new syntax is already covered in the RC 1 breaking change article at https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/implicit-namespaces-rc1. And since it's only enabled for new projects going forward, that's not considered a breaking change.

Reply:
I'll revert this file.